### ISSUE TITLE
luis export version followed by ludown refresh results in bad token error

### ISSUE THREAD

**Issue by srozga** [Microsoft/botbuilder-tools#167]
**Describe the bug**
I created a .luisrc file that connects to my LUIS model. I ran luis export > export.json to get the LUIS json down on to my computer. I did this in PowerShell. Next, I tried running ludown refresh -i export.json, but that resulted in the error:
```
PS C:\dev\LUIS> ludown refresh -i .\export.json
Oops! Something went wrong.
SyntaxError: Unexpected token � in JSON at position 0 

```

**To Reproduce**
1. Create .luisrc file as per docs.
2. Run `luis export version > export.json`
3. Run `ludown refresh -i export.json`

Thanks!

**Comment by vishwacsena** [comment]
Thanks for reporting this! Fixed in ludown 1.0.21.

**Comment by srozga** [comment]
works like a charm, thanks!
